crates/sparrow-main/tests/e2e/map_tests.rs

```diff
@@ -0,0 +1,7 @@
+//! Basic e2e tests for the map functions.
+
+use indoc::indoc;
+use sparrow_api::kaskada::v1alpha::TableConfig;
+use uuid::Uuid;
+
+use crate::{DataFixture, QueryFixture};
```

Comment:
This file seems mostly empty / unused? Is there any reason it exists (most tests are in `collection_tests.rs`? Should they move here?

Reply:
oops - removed file